.photofinish.toml

```diff
@@ -291,4 +288,40 @@ files = [
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_host_discovery.json",
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_sap_system_discovery.json",
     "./test/fixtures/scenarios/healthy-27-node-SAP-cluster/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_subscription_discovery.json",    
+]
+
+[NWD-00-present]
```

Comment:
I've noticed we usually start these with `sapsystem-` or `database-` or `host-`... depending on what we are pushing. In this case it's obvious but I'd add it to keep the good habit I guess :-)